packages/stable/src/types.ts

```diff
@@ -2763,3 +2766,167 @@ export interface EntityMatchingFilter {
 export interface EntityMatchingFilterRequest extends FilterQuery {
   filter?: EntityMatchingFilter;
 }
+
+export interface DiagramResponseBase extends ContextJobInfo {
+  /**
+   * Contextualization job ID.
+   */
+  jobId: ContextJobId;
+}
+
+export interface DiagramDetectConfig {
+  /**
+   * This field determines the string to search for and to identify object entities.
+   */
+  searchField?: string;
+  /**
+   * Allow partial (fuzzy) matching of entities in the engineering diagrams. Creates a match only when it is possible to do so unambiguously.
+   */
+  partialMatch?: string;
+  /**
+   * Each detected item must match the detected entity on at least this number of tokens. A token is a substring of consecutive letters or digits.
+   */
+  minTokens?: number;
+}
+
+type OneOfFiledId =
```

Comment:
```suggestion
type FileIdEither =
```
better to use postfix instead of prefix, also change it so it's more consistent with `IdEither` type